src/main-process/tray.js

```diff
@@ -113,13 +128,43 @@ class IpcManager {
             });
         }
 
-        this.tray.setContextMenu(Menu.buildFromTemplate([...this.playToggle, ...this.menu]));
+        this._setContextMenu('play');
     }
 
     hide() {
 
         this.tray.destroy();
     }
+
+    _setContextMenu(state) {
+        const playPauseItem = state === 'pause' ? this.pauseToggle : this.playToggle;
+        this.tray.setContextMenu(Menu.buildFromTemplate([...this.songDetails, ...playPauseItem, ...this.menu]));
+    }
+
+
+    _updateTrayMetadata(metadata) {
+        this.songDetails = [
+            {
+                label: `Song: ${metadata.title}`,
+                enabled: false
+            },
+            {
+                label: `Artist: ${metadata.artist}`,
+                enabled: false
+            },
+            {
+                label: `Album: ${metadata.album}`,
+                enabled: false
+            },
+            {
+                type: 'separator'
+            }
+        ];
+        // this.songDetails[0].label = this._trayString(metadata);
+
+        // return `${metadata.title} - ${metadata.artist} - ${metadata.album}`;
```

Comment:
Yep. At first, I tried putting all the metadata on one line, but it didn't look very good. I'll remove it